cmd/poseidon/main.go

```diff
@@ -37,6 +38,33 @@ func shutdownSentry() {
 	}
 }
 
+func initProfiling(options *config.Profiling) (cancel func()) {
```

Comment:
I learned recently that it is not always faster to pass the arguments as a pointer, especially for small structs. This is due to the stack / heap management. If it catches your interest, take a look [here](https://medium.com/a-journey-with-go/go-introduction-to-the-escape-analysis-f7610174e890), for example.

Reply:
Thanks for providing that link and the information; I changed the occurrence here.